app/commands/utils.py

```diff
@@ -1,3 +1,9 @@
+def log_ops_message(client, message):
+    channel_id = "C0388M21LKZ"
```

Comment:
What is this channel ID?

Reply:
Something that shouldn't change and is hardcoded - or I am too lazy to make it an env variable because realistically it won't ever change